homeassistant/components/vallox/sensor.py

```diff
@@ -187,6 +217,23 @@ class ValloxSensorEntityDescription(SensorEntityDescription):
         state_class=SensorStateClass.MEASUREMENT,
         native_unit_of_measurement=TEMP_CELSIUS,
     ),
+    ValloxSensorEntityDescription(
```

Comment:
Should this have an entity category?

Reply:
We have "diagnostic" everywhere in ValloxSensorEntity class.